src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/custom.py

```diff
@@ -150,15 +189,18 @@ def update_app_service_plan(resource_group, name, tier=None, number_of_workers=N
 def _get_sku_prefix(tier):
     return 'D' if tier.lower() == 'shared' else tier[0:1]
 
-def _get_sku_name(tier, worker_size):
-    worker_size_mappings = {
-        'small': '1',
-        'medium': '2',
-        'large': '3',
-        'extralarge': '4'
-    }
-    sku = _get_sku_prefix(tier) + worker_size_mappings[worker_size.lower()]
-    return sku
+def _get_sku_name(tier):
+    tier = tier.upper()
+    if tier in ['FREE', 'SHARED']:
+        return tier
+    elif tier in ['B1', 'B2', 'B3']:
+        return 'BASIC'
+    elif tier in ['S1', 'S2', 'S3']:
+        return 'STANDARD'
+    elif tier in ['P1', 'P2', 'P3']:
+        return 'PREMIUM'
+    else:
```

Comment:
Should we open an issue against their Swagger? @johanste

---
In reply to: [79504330](https://github.com/Azure/azure-cli/pull/924#discussion_r79504330) [](ancestors = 79504330)